packages/discord.js/src/util/Transformers.js

```diff
@@ -92,8 +92,29 @@ function _transformAPIIncidentsData(data) {
   };
 }
 
+/**
+ * Transforms a collectibles object to a camel-cased variant.
+ *
+ * @param {APICollectibles} collectibles The collectibles to transform
+ * @returns {Collectibles}
+ * @ignore
+ */
+function _transformCollectibles(collectibles) {
```

Comment:
Why is this needed? Looks like it's effectively just toSnakeCase(), there's no real transformations of the data happening.

Reply:
It's not toSnakeCase, it's the other way around, and we have similar functions for other transforms